wscript

```diff
@@ -101,33 +102,34 @@ def configure(conf):
 def build(bld):
     libndn_svs = dict(
         target='ndn-svs',
-        vnum=VERSION,
-        cnum=VERSION,
```

Comment:
`vnum` and `cnum` are meaningful for shared libraries only, hence moved to `shlib` below.